site/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -28,19 +28,25 @@ export const useStyles = makeStyles((theme) => ({
   },
 }))
 
+interface LocationState {
+  isRedirect: boolean
+}
+
 export const LoginPage: React.FC = () => {
   const styles = useStyles()
   const location = useLocation()
   const xServices = useContext(XServiceContext)
   const [authState, authSend] = useActor(xServices.authXService)
   const isLoading = authState.hasTag("loading")
   const redirectTo = retrieveRedirect(location.search)
+  const locationState = location.state ? (location.state as LocationState) : null
```

Comment:
Do we need to use a ternary because of types? Or can we:
```suggestion
  const locationState = location.state ??  null
```

Reply:
`location.state` can be `null`, but since we are typecasting it, the compiler thinks it is never `null`, so we are typecasting it only if it is not null.